homeassistant/components/comelit/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""Comelit integration."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PIN,
+    EVENT_HOMEASSISTANT_STOP,
+    Platform,
+)
+from homeassistant.core import Event, HomeAssistant
+
+from .const import _LOGGER, DOMAIN
+from .coordinator import ComelitSerialBridge
+
+PLATFORMS = [Platform.LIGHT]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Comelit platform."""
+    _LOGGER.debug("Setting up Comelit component")
```

Comment:
I'd suggest to remove this because setting up integrations is an info level print anyway

Reply:
Done